src/main/java/org/thoughtcrime/securesms/notifications/NotificationCenter.java

```diff
@@ -366,7 +368,28 @@ public void notifyReaction(int accountId, int contactId, int msgId, String react
 
         DcContact sender = dcContext.getContact(contactId);
         String shortLine = context.getString(R.string.reaction_by_other, sender.getDisplayName(), reaction, dcMsg.getSummarytext(2000));
-        maybeAddNotification(accountId, dcContext.getChat(dcMsg.getChatId()), msgId, shortLine, shortLine, false);
+        maybeAddNotification(accountId, dcContext.getChat(dcMsg.getChatId()), msgId, shortLine, shortLine, true);
```

Comment:
the "play in chat sound" needs to be false here and for webxdc-info.

at some point, we can factor that out , but well ...